api/_routers/03-host_detection.go

```diff
@@ -45,7 +44,6 @@ func (h *HostRouter) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	host, _, err := net.SplitHostPort(raddr)
 	if err != nil {
 		logrus.Error(err)
```

Comment:
if we're not reporting it to sentry then we also shouldn't be logging it at the error level. Please reduce this to debug.